dll/win32/comctl32/comctl32.manifest

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0">
+  <assemblyIdentity type="win32" name="Microsoft.Windows.Common-Controls" version="6.0.2600.2982" processorArchitecture="" publicKeyToken="6595b64144ccf1df"/>
```

Comment:
`processorArchitecture=""`, or `processorArchitecture="*"`  ?
Also, isn't this file going to be redundant with
https://git.reactos.org/?p=reactos.git;a=blob;f=dll/win32/comctl32/x86_microsoft.windows.common-controls_6595b64144ccf1df_6.0.2600.2982_none_deadbeef.manifest;h=0e253694b7390c5f81e136c34ff6ff87917cbc99;hb=HEAD
and
https://git.reactos.org/?p=reactos.git;a=blob;f=dll/win32/comctl32/amd64_microsoft.windows.common-controls_6595b64144ccf1df_6.0.2600.2982_none_deadbeef.manifest;h=6aae024df59c94ba95968b7e70ed10cc5384f5b1;hb=HEAD
as well as:
https://git.reactos.org/?p=reactos.git;a=blob;f=dll/win32/comctl32/x86_microsoft.windows.common-controls_6595b64144ccf1df_5.82.2600.2982_none_deadbeef.manifest;h=667503808bb0cf7cc3e0685150b71d6185f799d6;hb=HEAD
and
https://git.reactos.org/?p=reactos.git;a=blob;f=dll/win32/comctl32/amd64_microsoft.windows.common-controls_6595b64144ccf1df_5.82.2600.2982_none_deadbeef.manifest;h=f272350fdfd2baa241d1c56e1ac01e251da684da;hb=HEAD
??
(Note that the last two manifests are probably for some sort of comctl32 version 5 compatibility, with the "versionining = no" addition.)

Reply:
this is actually a mistake include,
We do the manifest in a way more compatible way atm.

Reply:
Please check that the list of controls listed in the manifest hasn't changed in our files vs. that wine manifest.